### PR TITLE
GCE: add an option to disable creation of Windows nodes.

### PR DESCRIPTION
**Pull request by oxddr**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add ability to disable creation of Windows nodes. In most scalability tests they are not create anyway. This saves us like 3s ;)

/sig scalability
/assign @mm4tt 

```release-note
NONE
```

### CODE REVIEW

**Comment by oxddr** [comment]
PTAL

**Comment by mborsz** [comment]
/lgtm

LGTM, but I don't have approval powers here.

File: cluster/gce/config-default.sh [hunk 1]
```diff
@@ -481,5 +481,8 @@ WINDOWS_NODE_TAINTS="${WINDOWS_NODE_TAINTS:-node.kubernetes.io/os=win1809:NoSche
 # Whether to set up a private GCE cluster, i.e. a cluster where nodes have only private IPs.
 GCE_PRIVATE_CLUSTER="${KUBE_GCE_PRIVATE_CLUSTER:-false}"
 
+# Whether to enable Windows nodes. Enabled by default.
+GCE_WINDOWS_NODES="${KUBE_GCE_WINDOWS_NODES:-true}"
```
Comment (wojtek-t) [inline cluster/gce/config-default.sh]: Why do we need it? If you set NUM_WINDOWS_NODES=0 it also doesn't create windows nodes?

Comment (oxddr) [reply cluster/gce/config-default.sh]: This PR is from some time ago and IIRC this wasn't the case. But I'll have a look to double check that.

Comment (oxddr) [reply cluster/gce/config-default.sh]: It's actually not needed. Replaced the condition with NUM_WINDOWS_NODES.

**Comment by oxddr** [comment]
PTAL

File: cluster/gce/windows/node-helper.sh [hunk 2]
```diff
@@ -52,7 +52,9 @@ function get-windows-node-instance-metadata {
 # $1: template name (required).
 # $2: scopes flag.
 function create-windows-node-instance-template {
-  local template_name="$1"
-  local scopes_flag="$2"
-  create-node-template "${template_name}" "${scopes_flag}" "$(get-windows-node-instance-metadata-from-file)" "$(get-windows-node-instance-metadata)" "windows"
+  if [[ "${NUM_WINDOWS_NODES}" -gt "0" ]]; then
+    local template_name="$1"
+    local scopes_flag="$2"
+    create-node-template "${template_name}" "${scopes_flag}" "$(get-windows-node-instance-metadata-from-file)" "$(get-windows-node-instance-metadata)" "windows"
```
Comment (wojtek-t) [inline cluster/gce/windows/node-helper.sh]: What's the point of this change? The fact that we don't create windows nodes at the beginning doesn't mean we won't want to do that later...

Comment (oxddr) [reply cluster/gce/windows/node-helper.sh]: As stated in the PR description: I want to disable creation of Windows template. In scalability tests we don't use them at the moment and it takes some time to create it.

Comment (wojtek-t) [reply cluster/gce/windows/node-helper.sh]: Saving 3s is not worth it imho.

**Pull request closed without merging**